.env

```diff
@@ -0,0 +1 @@
+JWT_SECRET="json web token secrate key"
```

Comment:
This file should not be pushed to git. can you please add it to gitignore?

Reply:
ok